source/shared/cpp/ObjectModel/Column.cpp

```diff
@@ -112,4 +112,14 @@ std::shared_ptr<Column> Column::Deserialize(const Json::Value& value)
 std::shared_ptr<Column> Column::DeserializeFromString(const std::string& jsonString)
 {
     return Column::Deserialize(ParseUtil::GetJsonValueFromString(jsonString));
+}
+
+std::shared_ptr<BaseActionElement> Column::GetSelectAction() const
```

Comment:
I was going to add that in a separate PR, but I guess there really is no reason to split it out that much. Adding the fields to serialization logic now.